apps/oxlint/tsconfig.json

```diff
@@ -6,5 +6,9 @@
     "target": "ESNext",
     "noImplicitAny": true,
     "skipLibCheck": true
-  }
+  },
+  "exclude": [
+    "node_modules",
+    "fixtures"
+  ]
```

Comment:
To avoid TypeScript looking in `fixtures`, which contains some erroneous TS code in tsgolint tests.